devices/jdwhale.js

```diff
@@ -0,0 +1,54 @@
+const utils = require('zigbee-herdsman-converters/lib/utils');
+const tz = require('zigbee-herdsman-converters/converters/toZigbee');
+const exposes = require('zigbee-herdsman-converters/lib/exposes');
+const e = exposes.presets;
+const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
+const reporting = require('zigbee-herdsman-converters/lib/reporting');
+
+module.exports = [
```

Comment:
Are these the following devices? If yes the vendor looks to be `J.XUAN`? https://www.jd.id/product/j-xuan-prz01-zigbee-intelligent-human-body-sensor-mobile-sen_626005876/626005877.html

Reply:
Yes, it may need to be renamed j.xuan